import.py

```diff
@@ -904,7 +904,11 @@ def main(arg_list: List[str]) -> None:
                 json_data: Dict[str, str] = json.loads(resp)
                 if "slug" in json_data:
                     slug = json_data["slug"]
-                    print(f"https://decomp.me/scratch/{slug}")
+                    token = json_data["claim_token"]
```

Comment:
Do you want `json_data.get("token")` so it'll work before the decomp.me change is deployed?

Reply:
```suggestion
                    token = json_data.get("claim_token")
```

Reply:
Thanks, I always forget about KeyError in Python 😅